tests/protocols/test_websocket.py

```diff
@@ -968,7 +968,7 @@ async def get_subprotocol(url: str):
         (MAX_WS_BYTES, MAX_WS_BYTES, 0),
         (MAX_WS_BYTES_PLUS1, MAX_WS_BYTES, 1009),
         (10, 10, 0),
-        (11, 10, 1009),
+        (32, 10, 1009),
```

Comment:
@Kludex I am facing strange situation, in this test case on keeping the message size greater than 31 . Test is passing, but on values from 11 to 31 , it is failing. not sure, where there is some issue with my code, or ServerProtocol or with the client.

Reply:
I can, but it will take some weeks. I'm sorry. I'm a bit busy.

Reply:
sure, whenever the time permits you :)

Reply:
Hi, actually I was able to find the reason for it. 
In the test just like, There max_size limit on the server. There were also max_size limit on the client side also
like this

async with websockets.client.connect(
            url, max_size=client_size_sent
   ) as websocket:
   
   So, in case when client_size_sent is 11, The close frame is being received with code 1009, along with a reason as "over size limit ({length} > {max_size} bytes)". Now the total size of this close which include the close code and the reason is summing more than 11. So, before the client processes this close frame, it is detecting the payload is bigger than 11 bytes. which is causing the client to close the connection on its end without processing the close frame from the server.
   
 So, for this test to work correctly, I have changed the client initiation logic from 
        async with websockets.client.connect(
            url, max_size=client_size_sent
        ) as websocket:

to this latest commit

        async with websockets.client.connect(url, max_size=MAX_WS_BYTES) as websocket:

Because, anyway, we want to test that atleast server is able send the close frame correctly